CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+Contributing
+============
+
+We welcome bug fixes and patches from 3rd party contributors. Please
+see the [Contributor Agreement](https://www.arangodb.com/community#contribute)
+for details.
+
+Please follow these guidelines if you want to contribute to ArangoDB:
+
+Reporting Bugs
+--------------
+
+When reporting bugs, please use our issue tracker on GitHub.  Please make sure
+to include the version number of ArangoDB in your bug report, along with the
```

Comment:
Should this be ArangoDB & go-driver version?

Reply:
I've include commit hash (go-driver has not "normal" version number)